target/arc/semfunc-mapping.def

```diff
@@ -782,6 +787,11 @@ MAPPING(fdmsub, FDMSUB, 4, 0, 1, 2, 3)
 MAPPING(fdnmadd, FDNMADD, 4, 0, 1, 2, 3)
 MAPPING(fdnmsub, FDNMSUB, 4, 0, 1, 2, 3)
 
+MAPPING(fsmaddv2, FSMADDv2, 3, 0, 1, 2)
```

Comment:
I don't really fancy this solution. Maybe there is a better one.

Reply:
As I said in the PR comment (in "Mnemonic collisions"), due to the different amount of arguments, and same name, I don't see any other non invasive way of doing this.
Do you see any other solution that doesn't involve changing the decoder? Current solution "just works" and is very simple.

Reply:
Indeed, there is no another way to solve it another way - at least HS4x and HS5x are mixed in one machine type and they cannot be decoupled by "defines". However, I would suggest to name those fused instructions for ARCv3 FPU as `FSMADDv3` and so on. Otherwise, it's not obvious for me what is `fsmaddv2` and what is `fsmadd`. Or at least document it somewhere in code.